modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -464,15 +464,23 @@ protected void testGetObjectDefinitionsPageWithFilter(
 		testGetObjectDefinitionsPage_addObjectDefinition(objectDefinition2);
 
 		for (EntityField entityField : entityFields) {
-			Page<ObjectDefinition> page =
-				objectDefinitionResource.getObjectDefinitionsPage(
-					null, null,
-					getFilterString(entityField, operator, objectDefinition1),
-					Pagination.of(1, 2), null);
+			_validateGetObjectDefinitionsPageWithFilter(
+				Collections.singletonList(objectDefinition1),
+				getFilterString(entityField, operator, objectDefinition1));
+
+			// Test reindex objectDefinition when update objectFolder
```

Comment:
I have a suggestion for this comment:
`// Test reindex objectDefinition when updating an objectFolder`

Does it makes sense?

Reply:
Done